language/lang.go

```diff
@@ -189,4 +189,12 @@ type GenerateResult struct {
 	// correspond. These values are passed to Resolve after merge. The type
 	// is opaque since different languages may use different representations.
 	Imports []interface{}
+
+	// RelsToIndex is a list of additional directories to index for dependency
+	// resolution, expressed as slash-separated paths relative to the repository
+	// root, or "" for the root directory itself. If indexing is enabled,
+	// libraries in these directories are indexed before dependencies are
```

Comment:
*in these directories **and parent directories**?

Reply:
No: the parent directories are configured (the `.Configure` method is called on each extension), but the callback to `Walk2` is not called.

This, together with @fmeum's comment above made me think more about this, change the logic a bit, and add a test. We might first visit and configure a directory as a parent directory. Later, something else could add it to `RelsToVisit`. The callback should then be called, but we shouldn't reconfigure it.